css/main.css

```diff
@@ -96,27 +100,38 @@ h1,
 h2,
 h3,
 h4,
-h5 {
-  font-family: var(--sans-serif-fonts);
-  line-height: 1.6;
+h5{
+  font-family: "Lato", sans-serif;
+  line-height: 2.5rem;
   margin: 0.66em 0;
   font-weight: bold;
-}
 
-h1 {
-  font-size: 1.6em;
 }
 
-h2 {
-  font-size: 1.3em;
+/* STICKY HEADERS START */
+
+h1,
+h2.help-heading,
+h3.help-heading
```

Comment:
should help-specific stuff live in [help.css generated by gen_help_html.lua](https://github.com/neovim/neovim/blob/266a1c61b97313f2a516140fc691dd8be8eed47a/scripts/gen_help_html.lua#L841)? else things get hard to track down.

Reply:
Well, if this one is generated automatically then - yes.